subsys/usb/device/class/cdc_acm.c

```diff
@@ -1029,16 +1029,11 @@ static void cdc_acm_poll_out(const struct device *dev, unsigned char c)
 {
 	struct cdc_acm_dev_data_t * const dev_data = dev->data;
 
-	if (!dev_data->configured || dev_data->suspended) {
-		LOG_INF("USB device not ready, drop data");
-		return;
-	}
-
 	dev_data->tx_ready = false;
 
 	while (!ring_buf_put(dev_data->tx_ringbuf, &c, 1)) {
 		if (k_is_in_isr() || !dev_data->flow_ctrl) {
-			LOG_INF("Ring buffer full, discard %c", c);
+			/* Ring buffer full, discard character. */
```

Comment:
This LOG_INF must stay. If this bothers someone in the application, then USB_CDC_ACM_LOG_LEVEL should be disabled or the level set higher.

Reply:
You don't understand the implications of that.